client/blocks/importer/wordpress/upgrade-plan/style.scss

```diff
@@ -211,4 +226,182 @@ $business-plan-color: #7f54b3;
 			fill: $business-plan-color;
 		}
 	}
+
+	.import__upgrade-plan-hosting-details {
+		max-width: 100%;
+		background-color: #fbfbfb;
+		padding: 1.5rem;
+		border-top: 0;
+		border-left: 1px solid #e0e0e0;
+		border-right: 1px solid #e0e0e0;
+		border-bottom: 1px solid #e0e0e0;
+		border-top-right-radius: 0;
+		border-bottom-right-radius: 4px;
+		border-bottom-left-radius: 4px;
+		display: flex;
+		flex-direction: column;
+		margin-top: 0;
+		margin-bottom: 2.75rem;
+		gap: 2rem;
+
+		@include break-medium {
+			margin-right: 0;
+			margin-left: 0;
+			margin-top: 0.75rem;
+			border-top: 1px solid #e0e0e0;
+			border-top-right-radius: 4px;
+			border-bottom-right-radius: 4px;
+			border-bottom-left-radius: 0;
+			border-left: 0;
+			width: 324px;
+			padding: 2.75rem 1.5rem;
+		}
+
+		.import__upgrade-plan-hosting-details-header {
+			text-align: center;
+			.import__upgrade-plan-hosting-details-header-main {
+				font-size: 1.25rem;
+				color: var(--studio-gray-100, #101517);
+				margin-bottom: 0.5rem;
+				font-family: Recoleta, "Noto Serif", Georgia, "Times New Roman", Times, serif;
+				line-height: normal;
+			}
+
+			.import__upgrade-plan-hosting-details-header-subtext {
+				color: var(--studio-gray-50, #646970);
+				font-size: 0.75rem;
+				line-height: 16px;
+				margin-bottom: 0;
+			}
+		}
+
+		.import__upgrade-plan-hosting-details-list {
+			font-size: 0.875rem;
+			ul {
+				list-style: none;
+				margin: 0;
+			}
+
+			li {
+				display: flex;
+				gap: 0.5rem;
+				margin-bottom: 1rem;
+
+				&:last-child {
+					margin-bottom: 0;
+				}
+			}
+
+			.import__upgrade-plan-hosting-details-list-icon {
+				border-radius: 4px;
+				background-color: #f2e9ed;
+				padding: 4px;
+				border: 2px solid #f2e9ed;
+				fill: #984a9c;
+			}
+
+			.import__upgrade-plan-hosting-details-list-stats {
+				display: flex;
+				flex-direction: column;
+				gap: 0.125rem;
+				line-height: 16px;
```

Comment:
Can be `1rem`

Reply:
We cannot user rem in `line-height`, so I had to use 1.4 as value instead.

Reply:
Ough, ok. I didn't see that it was the line-height property. 👍